src/components/ApplicationListView/ApplicationList.tsx

```diff
@@ -41,25 +46,30 @@ const ApplicationList: React.FC = () => {
       +new Date(app2.metadata.creationTimestamp) - +new Date(app1.metadata.creationTimestamp),
   );
 
-  return (
-    <Page
-      heading="Applications"
-      customButton={
-        <Link className="pf-c-button pf-m-primary" to={'/app-studio/create'}>
-          Create Application
-        </Link>
-      }
-    >
-      {!allApplications || allApplications.length === 0 ? (
-        <EmptyState variant={EmptyStateVariant.full}>
-          <EmptyStateIcon icon={SearchIcon} />
-          <EmptyStateBody data-test="empty-state-body">
-            <p>No applications found</p>
-            <br />
-            <Link to={`/app-studio`}>Create an application</Link> to get started.
-          </EmptyStateBody>
-        </EmptyState>
-      ) : (
+  const content =
+    !allApplications || allApplications.length === 0 ? (
+      <EmptyState variant={EmptyStateVariant.full}>
+        <EmptyStateIcon icon={SearchIcon} />
+        <EmptyStateBody data-test="empty-state-body">
+          <p>No applications found</p>
+          <br />
+          <Link to={`/app-studio`}>Create an application</Link> to get started.
```

Comment:
Nit. I realize that the redirect would still take us to the create page in this case,
but I think it's better to be consistent with the `Button` url below.
```suggestion
          <Link to="/app-studio/create">Create an application</Link> to get started.
```